src/index.js

```diff
@@ -17,6 +25,12 @@ app.get('/', (req, res) => {
   res.status(200).json({ message: 'welcome' });
 });
 
+app.get('/api', (req, res) => {
```

Comment:
Please remove this endpoint, it is decreasing our coverage and we already have that endpoint.

Reply:
Noted. Thanks for the feedback.